client/scripts/models/IChainAdapter.ts

```diff
@@ -32,7 +32,6 @@ abstract class IChainAdapter<C extends Coin, A extends Account<C>> {
   public get serverLoaded() { return this._serverLoaded; }
 
   private async _postModuleLoad(listenEvents = false): Promise<void> {
-    await this.app.comments.refreshAll(this.id, null, CommentRefreshOption.LoadProposalComments);
```

Comment:
why is this removed now?

Reply:
We only want to fetch comments that belong to the fetched threads, so this kind of bulk call is undesirable.

Reply:
From spec:
> We should figure out if it's possible to get rid of the /bulkComments endpoint if comments can always be loaded alongside the thread they exist within.

Reply:
sounds good!

we fetch reactions too, right? could remove the commented-out next line?